apps/addons/models.py

```diff
@@ -485,7 +488,9 @@ def delete(self, msg='', reason=''):
             REASON GIVEN BY USER FOR DELETION: %(reason)s
             """ % context
             log.debug('Sending delete email for %(atype)s %(id)s' % context)
-            subject = 'Deleting %(atype)s %(slug)s (%(id)d)' % context
+            # Don't localize email to admins, use 'en-US' always.
+            with no_translation():
```

Comment:
actually this one isn't needed, it's only the type that is localized, and we already "un-localized" it above ;)